docs/sources/alerting/manage-notifications/webhook-notifier.md

```diff
@@ -110,12 +110,15 @@ Example JSON body:
 | annotations  | object | Annotations that are part of this alert, map of string keys to string values       |
 | startsAt     | string | Start time of the alert                                                            |
 | endsAt       | string | End time of the alert, default value when not resolved is `0001-01-01T00:00:00Z`   |
-| valueString  | string | Values that triggered the current status                                           |
+| valueString  | string | **Will be deprecated in Grafana 10**                                               |  
```

Comment:
Oh, I found TODO that we missed https://github.com/grafana/alerting/blob/e067f119be06c7ae1e225d8b4bfc2c832a9b23e3/templates/template_data.go#L129 @grobinson-grafana should we delete it now?

Reply:
Waiting for the response.

Reply:
Let's delete `valueString` from the doc at all. We will delete it in code after that.